services/brig/src/Brig/Index/Migrations.hs

```diff
@@ -120,14 +120,14 @@ failIfIndexAbsent targetIndex =
 
 -- | Runs only the migrations which need to run
 runMigration :: MigrationVersion -> MigrationActionT IO ()
-runMigration ver = do
-  vmax <- latestMigrationVersion
-  if ver > vmax
+runMigration expectedVersion = do
+  foundVersion <- latestMigrationVersion
+  if expectedVersion > foundVersion
     then do
       Log.info $
         Log.msg (Log.val "Migration necessary.")
-          . Log.field "expectedVersion" vmax
-          . Log.field "foundVersion" ver
+          . Log.field "expectedVersion" ver
```

Comment:
Shouldn't this be 

```suggestion
          . Log.field "expectedVersion" expectedVersion
```
? :thinking: 

(I cannot see where `ver` comes from.)

Reply:
Somehow I missed this 😅

Reply:
good thing we have sven and the compiler!  :)